cfme/common/provider.py

```diff
@@ -582,6 +582,7 @@ def get_paginator_value():
 class CloudInfraProvider(BaseProvider, PolicyProfileAssignable):
     vm_name = ""
     template_name = ""
+    volume_name = "Cloud volumes"
```

Comment:
There is no need for this here, the only reason you'd want to do this, is if the method was on the BaseProvider and you wanted to add specific volume name that differed between the various sub classes. Plus, it seems it is only relevant for Cloud, judging by the naming, in which case, putting it in the cloud module would be better. Having said all of this, I think we want to make this a navmazing destination, @mshriver what say you?